### PR TITLE
[TEST] Add converter and decoder subplugins test

### PR DESCRIPTION
**Pull request by gichan-jang**
Add tensor converter and decoder subplugin test
    
Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/nnstreamer_converter/unittest_converter.cc [hunk 1]
```diff
@@ -184,6 +185,157 @@ TEST (tensorConverterCustom, invalidParam3_n)
...
+ */
+TEST (tensorConverter, subpluginInvalidParam1_n)
+{
+  NNStreamerExternalConverter *sub = g_new0 (NNStreamerExternalConverter, 1);
+  ASSERT_TRUE (sub != NULL);
+
+  sub->name = NULL;
+  sub->query_caps = conv_query_caps;
+  sub->get_out_config = conv_get_out_config;
+  sub->convert = conv_convert;
```
Comment (dongju-chae) [inline tests/nnstreamer_converter/unittest_converter.cc:248]: It seems that codes are somewhat redundant. How about declaring common function returning `sub`?
For example,
```
NNStreamerExternalConverter *sub = get_default_converter ();
sub->name = NULL;
```
I think this approach is easy to understand what makes a failure.

Comment (gichan-jang) [reply tests/nnstreamer_converter/unittest_converter.cc]: Nice! Updated.

**Review by dongju-chae — APPROVED** [review]
LGTM

File: tests/nnstreamer_converter/unittest_converter.cc [hunk 2]
```diff
@@ -184,6 +185,124 @@ TEST (tensorConverterCustom, invalidParam3_n)
...
+}
+
+/**
+ * @brief Test for plugin registration
+ */
+TEST (tensorConverter, subpluginNoraml)
+{
+  NNStreamerExternalConverter *sub = get_default_external_converter ("mode");
+  EXPECT_TRUE (registerExternalConverter (sub));
+  g_free (sub);
```
Comment (jaeyun-jung) [inline tests/nnstreamer_converter/unittest_converter.cc:234]: Add code to unregister ext-converter.
OR
```
remove_default_external_converter (const gchar *name)
{
  var = _conv_find (name);
  if var
    removeextconverter (name);
}
```

Comment (gichan-jang) [reply tests/nnstreamer_converter/unittest_converter.cc:238]: Updated!

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**